src/features/wisdom/inlineSuggestions.ts

```diff
@@ -214,43 +174,53 @@ async function getInlineSuggestions(
   return inlineSuggestionUserActionItems;
 }
 
-export async function inlineSuggestionCommitHandler(
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  textEditor: vscode.TextEditor,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  edit: vscode.TextEditorEdit
-) {
+// Handlers
+
+export async function inlineSuggestionTriggerHandler() {
+  // This trigger handler is called when the user explicitly triggers inline suggestion through command
   if (vscode.window.activeTextEditor?.document.languageId !== "ansible") {
-    return [];
+    return;
   }
-  console.log("inlineSuggestionCommitHandler triggered");
   const editor = vscode.window.activeTextEditor;
   if (!editor) {
     return;
   }
+
+  // Trigger the suggestion explicitly
+  console.log("inlineSuggestion Handler triggered Explicitly");
+  vscode.commands.executeCommand("editor.action.inlineSuggest.commit");
```

Comment:
Looks like this will directly commit the suggestion and not be displayed as inline suggestion if yes this doesn't seem right.

Reply:
Thanks for pointing it out; it should be `editor.action.inlineSuggest.trigger.`